### ISSUE TITLE
Not working with Rematch 5

### ISSUE THREAD

**Issue by elmoonfire** [nihilistzsche/RematchElvUISkin#4]
**Description**
It appears this addon has a compatibility issue with the new Rematch 5 which was just released in their stable build channel.

**NihilistzscheUI Version**
1.3.0

**ElvUI Version**
13.47

**Steps to Reproduce**
Just loaded the game

**Expected Behavior**
No error and functioning

**Actual Behavior**
Error message below and the skin doesn't seem to be applied

**Mention what addons you are using.**
I'm using a lot of addons but the error message doesn't appear unless I use Rematch 5 instead of Rematch 4

**Additional Information**
Nothing additional

**Error Log**

```
Message: ...terface/AddOns/RematchElvUISkin/RematchElvUISkin.lua:605: attempt to call method 'ShowDialog' (a nil value)
Time: Tue Nov  7 17:47:28 2023
Count: 3
Stack: ...terface/AddOns/RematchElvUISkin/RematchElvUISkin.lua:605: attempt to call method 'ShowDialog' (a nil value)
[string "=[C]"]: in function `ShowDialog'
[string "@Interface/AddOns/RematchElvUISkin/RematchElvUISkin.lua"]:605: in function <...terface/AddOns/RematchElvUISkin/RematchElvUISkin.lua:599>

Locals: (*temporary) = <function> defined =[C]:-1

```

**Comment by MatthewRyanRead** [comment]
To add more info: The code appearing in the error tries to pop up an dialog window about the addon not working with Rematch 5, but that code itself does not work.  I tried removing that locally but, as expected, the addon still does not work.

I'm sure it will be a significant change to make this work with Rematch 5, unfortunately!

**Comment by Fleymour** [comment]
> I'm sure it will be a significant change to make this work with Rematch 5, unfortunately!

yeah new rematch broke elvui skin and tsmpetvalue .. hope those will update.. so lovly combination
